mk/cargo.sh

```diff
@@ -109,6 +109,11 @@ case $target in
       export CARGO_TARGET_X86_64_UNKNOWN_LINUX_MUSL_RUSTFLAGS="$rustflags_self_contained"
     fi
     ;;
+  loongarch64-unknown-linux-gnu)
+    export CC_loongarch64_unknown_linux_gnu=clang-$llvm_version
+    export CC_loongarch64_unknown_linux_gnu=llvm-ar-$llvm_version
```

Comment:
```suggestion
    export AR_loongarch64_unknown_linux_gnu=llvm-ar-$llvm_version
```

Reply:
Thanks. I will create a new PR to do this since I have lost authentication to the current PR repository.